.circleci/config.yml

```
@@ -162,7 +162,7 @@ jobs:
         - dependencies-v4-
     - run:
         name: Build (if needed)
-        command: test -e ios/build/gutenberg/Build/Products/Release-iphonesimulator/GutenbergDemo.app || SKIP_BUNDLING=true npm run core test:e2e:build-app:ios
+        command: test -e ios/build/gutenberg/Build/Products/Release-iphonesimulator/gutenberg.app || SKIP_BUNDLING=true npm run core test:e2e:build-app:ios
```

Just found an error that even with the cache hit, the build was always triggered. It's not related to this PR but maybe we can fix this as well.

```suggestion
        command: test -e gutenberg/packages/react-native-editor/ios/build/gutenberg/Build/Products/Release-iphonesimulator/gutenberg.app || SKIP_BUNDLING=true npm run core test:e2e:build-app:ios
```